encrypted-media/scripts/playback-persistent-license-events.js

```diff
@@ -0,0 +1,168 @@
+function runTest(config, testname) {
+
+    var testname = config.keysystem + ', successful playback, persistent-license, '
+                                    + /video\/([^;]*)/.exec( config.videoType )[ 1 ]
+                                    + ', set src before setMediaKeys, check events';
+
+    var configuration = {   initDataTypes: [ config.initDataType ],
```

Comment:
It just lines everything up with a tab position. Are smaller displays really a concern? This line is 20% of the display width on my laptop.